CHANGELOG.md

```diff
@@ -1,13 +1,16 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.2.2] - 2022-03-30
+### Fixed
+- Updated ECID-service dependency to fix parsing expiry values from AMCV cookie
+- Add ClientHints to Delivery API model in preparation for ClientHints support
```

Comment:
Are the client hints models included in 2.2.1?

Reply:
@dcottingham I've only removed the reference to ClientHints in `Context` DTO, but the actual `ClientHints` DTO added in #57 remains, so I think we should still mention it in the changelog..